src/base/rss/private/rss_parser.cpp

```diff
@@ -280,36 +280,39 @@ void Parser::parseRssArticle(QXmlStreamReader &xml)
 
     while (!xml.atEnd()) {
         xml.readNext();
+        const QString name(xml.name().toString());
 
-        if(xml.isEndElement() && xml.name() == "item")
+        if (xml.isEndElement() && (name == "item"))
             break;
 
         if (xml.isStartElement()) {
-            if (xml.name() == "title") {
-                article["title"] = xml.readElementText().trimmed();
+            const QString text(xml.readElementText().trimmed());
+            article[name] = text;
+
+            if (name == "title") {
+                article["title"] = text;
```

Comment:
This is outdated diff.
Now I rewritten the code to not re-assign fields.
But I still assign explicit Article fields explicitly to keep xml and our names independent.